packages/create-agents/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "create-agents",
+  "version": "0.1.0",
+  "description": "Create an Inkeep Agent Framework project",
+  "type": "module",
+  "bin": {
+    "create-agents": "./dist/index.js"
+  },
+  "files": [
+    "dist",
+    "templates"
+  ],
+  "scripts": {
+    "dev": "tsx src/index.ts",
+    "build": "tsc",
+    "test": "vitest",
```

Comment:
make sure pnpm test at project root doesn't throw errors because no tests exist for this package